www/css/style.css

```diff
@@ -135,4 +141,17 @@ background:url('../img/contact.background.jpg') no-repeat center center fixed;
   border: 0px;
 }
 
+.booking-button {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  border: 0px;
+  padding: 10px 10px 10px 10px;
+}
+
+h3 {
+  font-family: 'Julius Sans One', sans-serif;
+}
 
+.input-label {
+  font-family: 'Julius Sans One', sans-serif;
+}
```

Comment:
Well, if almost everything is one font, you can set it on the body and then override it where you need to - it just depends *how much* you are using each font. If you have a default font (it sounds like this one is the default) and just a different font on a header or something, it will save you a lot of code to set it on the body instead.